code/modules/assembly/bomb.dm

```diff
@@ -149,6 +149,7 @@
 	var/strength = 1
 
 	var/turf/ground_zero = get_turf(loc)
+	bomb_mixture.temperature += 10 //you might ask, why this? lol
```

Comment:
?

Reply:
strange, I could've swore I undid this when I changed the branch, hold on